packages/venia-ui/i18n/en_US.json

```diff
@@ -23,6 +23,9 @@
     "addressBook.headerText": "Change Shipping Information",
     "addressBookPage.addAddressText": "Add an Address",
     "addressBookPage.addressBookText": "Address Book",
+    "addressBookPage.addDialogTitle": "New Address",
+    "addressBookPage.editDialogTitle": "Edit Address",
+    "addressBookPage.makeDefaultAddress": "Make this my default address",
```

Comment:
Do these things need to be added to the french language pack too?

Reply:
Good idea to do so, so it doesn't get out of date.